src/app/questionGenerator.js

```diff
@@ -0,0 +1,44 @@
+class QuestionGenerator {
```

Comment:
If you would want to use this class outside of this module, what is necessary to do so?